include/crm/pengine/status.h

```diff
@@ -94,9 +94,15 @@ pe_rsc_is_anon_clone(const pe_resource_t *rsc)
  * \return true if resource is part of a bundle, false otherwise
  */
 static inline bool
-pe_rsc_is_bundled(pe_resource_t *rsc)
+pe_rsc_is_bundled(const pe_resource_t *rsc)
```

Comment:
The dependencies were a little tricky, not worried about it

Reply:
No worries on the move/drop. I was thinking to at least use `pe__const_top_resource()` in `pe_rsc_is_bundled()` once we have it, instead keeping the reimplementation that we added in the interim.

Low priority if we choose to

Reply:
Nope, pe_rsc_is_bundled() is public while pe__const_top_resource() isn't. I think the new implementation is more readable anyway, it's never been clear that `uber_parent()->parent != NULL` means it's a bundle

Reply:
I'd like to eventually add a non-const equivalent pe__top_resource() to get rid of the rest of those uses

Reply:
OH, because it's inlined, got it.

I agree the new implementation is more readable than the old one. It took me a couple minutes to make sure the old one was doing what it was supposed to do. I only meant let's use `pe__const_top_resource()` in the new implementation and compare its result against `pe_container`. That won't work because of the public inline.